travis-dependencies.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
 if [[ $TRAVIS_OS_NAME == "osx" ]] ; then
+  brew install ccache                       # enable caching on mac
+  PATH=/usr/local/opt/ccache/libexec:$PATH  # https://docs.travis-ci.com/user/caching/#ccache-on-OS-X
```

Comment:
I suppose this is not correct, what we need is `export PATH="/usr/local/opt/ccache/libexec:$PATH"` for it to actually have an effect on the following commands

Also, can you move these two lines after the "brew update" step?